node/Vagrantfile

```diff
@@ -0,0 +1,38 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+    # Start with base vagrant box. This one is only 271 MB in size.
+    config.vm.box = "minimal/trusty64"
+
+    # Create forwarding ports for client-guest machine access via localhost.
+    config.vm.network :forwarded_port, guest: 80, host: 7331
```

Comment:
Put auto-correct as `true`. This allows for automatic increment by 1 if a post is occupied.